platform-tests/src/platform/performance/load_test.go

```diff
@@ -69,22 +68,22 @@ var _ = Describe("Load performance", func() {
 	AfterEach(func() {
 		Expect(cf.Cf("delete", appName, "-f", "-r").Wait(testConfig.DefaultTimeoutDuration())).To(Exit(0))
 	})
+
 	Context("without HTTP Keep-Alive", func() {
 		It("has a response latency within our threshold", func() {
-			metrics, latency := loadTest(appName, loadTestRate, loadTestDuration, false)
+			metrics := loadTest(appName, loadTestRate, loadTestDuration, false)
 			generateJsonReport(metrics, "load-test-no-keep-alive.json")
 			vegeta.NewTextReporter(metrics).Report(os.Stdout)
-			Expect(time.Duration.Nanoseconds(latency)).To(BeNumerically("<", loadTestLatency))
```

Comment:
I'm not sure I understand what this code was doing. `time.Duration` is a type, not an instance of a type, so this doesn't make sense to me. Experimenting in the playground shows that this converts the Duration into an int64 number of nanoseconds, but I still don't understand how.

The newer version is much clearer.